packages/web-config-server/src/apiV1/dataBuilders/modules/covid-samoa/totalNumOfPassengerByFlightDate.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Tupaia Config Server
+ * Copyright (c) 2019 Beyond Essential Systems Pty Ltd
+ */
+import moment from 'moment';
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import { Flight, getTotalNumPassengers, FLIGHT_DATE } from './flight';
+
+export class TotalNumOfPassengerByFlightDate extends DataBuilder {
+  async build() {
+    const dataElementCodes = [FLIGHT_DATE];
+    const { startDate, endDate } = this.query;
+    this.query.startDate = undefined;
```

Comment:
Out of curiosity why are these being set to undefined?

Reply:
Sorry I forgot to comment that, usually when fetching result, `.fetchEvents()` will use startDate and endDate as a period to fetch events.

But here is a special case that, we want all events sending back as result, and then filter by selected period on `FLIGHT_DATE`, because `FLIGHT_DATE` is not the response time, is a data element.

Reply:
Fair enough, assigning unassigned feels strange to me, but as this is a custom builder in the covid model should be ok if it works.

Reply:
Maybe add a comment?

Reply:
Yeah good point, will leave a comment there.